tests/test_acceptance.py

```diff
@@ -83,6 +82,13 @@ def purge_output_dirs():
                 id="Nepal project",
                 marks=pytest.mark.skip(reason="WPF Nepal test directory not presnt"),
             ),
+            pytest.param(
+                "bolivia",
+                id="Bolivia project",
+                marks=pytest.mark.skip(
+                    reason="Bolivia project test directory not presnt"
+                ),
```

Comment:
Please add this test data in `https://repos.deltares.nl/repos/GFS_incubator/branches/ra2ce_test_data`.
You can still test this locally by checking out the `svn` repository in `ra2ce/tests/test_external_data`. This directory is excluded from `git` so it won't be picked up during version control changes.